src/common/exif.cc

```diff
@@ -502,6 +504,19 @@ static void _find_datetime_taken(Exiv2::ExifData &exifData, Exiv2::ExifData::con
   }
 }
 
+static void mat3mul(float *dst, const float *const m1, const float *const m2)
```

Comment:
at some point we will want to have a way to share this, we have mat3mul in many places now!